config/config.exs

```diff
@@ -56,6 +56,9 @@ config :bors, BorsNG.Endpoint,
   render_errors: [view: BorsNG.ErrorView, accepts: ~w(html json)],
   pubsub: [name: BorsNG.PubSub, adapter: Phoenix.PubSub.PG2]
 
+# Configure hooks when local merge is enabled
+config :bors, BorsNG.GitHub.Merge.Hooks, hooks_dir: {:system, "HOOKS_DIR", ".bors-hooks"}
```

Comment:
The comment doesn't make sense to me here. There's nothing that describes a local merge in this line?

Reply:
shell script hooks are only used when local merge is enabled; this lets a deployment configure where those hooks are stored in a repo.

this is another config value that might be better as a per-project config value in `bors.toml` though